.github/policies/resourceManagement.yml

```diff
@@ -364,8 +357,8 @@ configuration:
           label: 'Needs: Author Feedback'
       - removeLabel:
           label: 'Needs: Triage :mag:'
-      description: 
-    - if:
+    - description: Mark new issues as coming from Facebook when the author is from Facebook
```

Comment:
Meta?

Reply:
If and when we rename the label the description can be renamed too.